### ISSUE TITLE
Should the web design standards theme be minimalistic out of the box?

### ISSUE THREAD

**Issue by AlexBrown-appnovation** [18F/web-design-standards-drupal-d7#18]
Lots of themes out there come with a predefined page template including some sidebars, background colours, etc - but isn't the nature of this theme to be a blank slate?

I just want to start a discussion here before we start thinking about what kind of first impression we want to give users when they install the theme.

Alternatively, we could try and mimic something like the playbook home page to spruce it up a bit?

**Comment by fureigh** [comment]
Thanks for starting this conversation, @AlexBrown-appnovation!

I'm inclined to give users something that doesn't get in their way if they want to immediately apply it to their existing site. Sidebars should display if and only if the user has enabled them in the theme settings, for instance.

As for colors, perhaps we should offer a standards-consistent range of options in the theme settings? I'd love to hear what prospective users think about this.

**Comment by fureigh** [comment]
@jonraedeke and @pjurquilla, thanks for [expressing your interest in helping](https://github.com/18F/web-design-standards/issues/852)! Do you have thoughts on this discussion?

**Comment by smittyjd23** [comment]
I don't have an opinion but want to call out @grasmash or @jenter in case they do

**Comment by jenter** [comment]
@AlexBrown-appnovation @fureigh
I was going to give my 2 cents since we recently started working on the d7 version. It would be nice if there was the normal sidebar scenarios (body.sidebar-both,body.sidebar-left,body.sidebar-right) for a fresh install. After we installed this was the first thing we had to do. I can't imagine it would be viewed as not bare-bones with those things. We also created a new panel layout following the grid as well. The basics of the containers with the sidebars or panel layout columns are very easy to reference and/or remove when first installing 

CC @smittyjd23 @grasmash

**Comment by jonraedeke** [comment]
@AlexBrown-appnovation @fureigh
I agree with @jenter. It should come with some basic sidebar functionality. A few panel layouts would be convenient too and wouldn't get in the way for non-panels users.

Later, a subtheme of this with color module support (preloaded with good combinations) and more prescriptive layout would be a nice way to get non-SASS folks up and running for quick projects.

**Comment by AlexBrown-appnovation** [comment]
Ok some basic sidebar functionality is a great idea, @jenter if you've already got some stuff ready to share, feel free to PM me about it!

@jenter I may need to lean on you to contribute back your panel layout as that would be great to have out of the box.

@jonraedeke I'm feeling that color module support might have to come later on as a "nice to have" in a future phase, perhaps we write an issue for this and label it accordingly? There are so many great ideas and features we'd love to get into this first release, but we've got to put a box around the requirements to keep our eye on the finish line if you know what I mean.

**Comment by patrickcate** [comment]
Sidebar regions are pretty standard in most Drupal themes. A good baseline set of regions are those the Bartik and Seven core themes provide.

Some Drupal themes are made with the main intention of providing developers a base theme to create sub-themes from. Others are mainly turn-key themes to be used as is. Is the plan for this theme to lean towards one or the other?
